services/payments/tests/unit/conftest.py

```diff
@@ -24,15 +26,72 @@
     PaymentInitiated,
 )
 
+#
+# rabbit-MQ
+#
+
 
 @pytest.fixture
 def disable_rabbitmq_and_rpc_setup(mocker: MockerFixture) -> Callable:
-    def _doit():
-        # The following moduls are affected if rabbitmq is not in place
+    def _do():
+        # The following services are affected if rabbitmq is not in place
         mocker.patch("simcore_service_payments.core.application.setup_rabbitmq")
         mocker.patch("simcore_service_payments.core.application.setup_rpc_api_routes")
 
-    return _doit
+    return _do
+
+
+@pytest.fixture
+def mock_patch_setup_rabbitmq_and_rpc(disable_rabbitmq_and_rpc_setup: Callable):
+    disable_rabbitmq_and_rpc_setup()
+
+
+#
+# postgres
+#
+
+
+@pytest.fixture
+def disable_postgres_setup(mocker: MockerFixture) -> Callable:
+    def _setup(app: FastAPI):
+        app.state.engine = (
+            Mock()
+        )  # NOTE: avoids error in api._dependencies::get_db_engine
+
+    def _do():
+        # The following services are affected if postgres is not in place
+        mocker.patch(
+            "simcore_service_payments.core.application.setup_postgres",
+            spec=True,
+            side_effect=_setup,
+        )
+
+    return _do
+
+
+@pytest.fixture
+def mock_patch_setup_postgres(disable_postgres_setup: Callable):
+    disable_postgres_setup()
+
+
+@pytest.fixture
+def wait_for_postgres_ready_and_db_migrated(postgres_db: sa.engine.Engine) -> None:
+    """
+    Typical use-case is to include it in
+
+    @pytest.fixture
+    def app_environment(
+        ...
+        postgres_env_vars_dict: EnvVarsDict,
+        wait_for_postgres_ready_and_db_migrated: None,
+    )
+    """
+    assert postgres_db
+
+
+#
+# app
+#
 
```

Comment:
I find these comments not very useful... app for a fixture called app... what does this bring? same with the others. if the function naming is clear which it is, what is the added benefit here?

Reply:
again it brings sanity to me :-)
I use these as separators to group fixtures in categories. This ws the app category as in "app related fixtures" .Now we just have one, which coincidentally is called app